projects/NodeBB/default.nix

```diff
@@ -27,22 +27,29 @@
     };
   };
 
-  nixos.modules.services = {
-    nodebb = {
-      name = "NodeBB";
-      module = ./services/nodebb/module.nix;
-      examples = {
-        postgresql = {
-          module = ./services/nodebb/examples/postgresql.nix;
-          description = "";
-          tests.postgresql = import ./services/nodebb/tests/postgresql.nix args;
-        };
-        redis = {
-          module = ./services/nodebb/examples/redis.nix;
-          description = "";
-          tests.redis = import ./services/nodebb/tests/redis.nix args;
+  nixos = {
+    modules.services = {
+      nodebb = {
+        name = "NodeBB";
+        module = ./services/nodebb/module.nix;
+        examples = {
+          postgresql = {
+            module = ./services/nodebb/examples/postgresql.nix;
+            description = "";
+            tests.postgresql = import ./services/nodebb/tests/postgresql.nix args;
+          };
+          redis = {
+            module = ./services/nodebb/examples/redis.nix;
+            description = "";
+            tests.redis = import ./services/nodebb/tests/redis.nix args;
+          };
         };
       };
     };
+    demo.vm = {
+      module = ./services/nodebb/examples/postgresql.nix;
```

Comment:
We should probably add an `openPorts` option.

```nix
    networking.firewall.allowedTCPPorts = [ 4567 ];
```

**PS:** The service takes some time to start. We probably want a way to handle this in the future, which could be as simple as warning the users or displaying a live feed or something.